### ISSUE TITLE
I got an issue when try to send request with the backconnect proxy 

### ISSUE THREAD

**Issue by NoahJoe22** [Theodikes/ipv6-proxy-server#32]
root@check:~# curl -v -x http://154.6.110.10:30000 -U username:password http://ipinfo.io/ip
*   Trying 154.6.110.10:30000...
* Connected to 154.6.110.10 (154.6.110.10) port 30000 (#0)
* Proxy auth using Basic with user 'username'
> GET http://ipinfo.io/ip HTTP/1.1
> Host: ipinfo.io
> Proxy-Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
> User-Agent: curl/7.88.1
> Accept: */*
> Proxy-Connection: Keep-Alive
>
* HTTP 1.0, assume close after body
< HTTP/1.0 502 Bad Gateway
< Connection: close
< Content-type: text/html; charset=utf-8
<
<html><head><title>502 Bad Gateway</title></head>
<body><h2>502 Bad Gateway</h2><h3>Host Not Found or connection failed</h3></body></html>
* Closing connection 0

**Comment by Temporalitas** [comment]
![image](https://github.com/user-attachments/assets/5c67353b-7916-4887-a5ee-9c9c19bfa8a9)

Did you do all these steps? Is curl via proxy IPv6 working on your server?

**Comment by NoahJoe22** [comment]
yes I have tried all these steps and not working , even though there is no logs

**Comment by Temporalitas** [comment]
> yes I have tried all these steps and not working , even though there is no logs

No log file? It's impossible, if script was executed
